eslint.config.mjs

```diff
@@ -3,7 +3,7 @@ import eslintConfigPrettier from 'eslint-config-prettier';
 import globals from 'globals';
 
 export default [
-  { languageOptions: { globals: globals.node } },
+  { languageOptions: { globals: { ...globals.node, ...globals.browser } } },
```

Comment:
Unfortunately, new ESLint does not support nested comfigs, so I need to set web configuration here even tho only the html generator uses web APIs

Reply:
Are you actually using a browser to run the generator's code? Otherwise this is wrong and I would ask why you need it?

In any case, new ESLint supports parts of configs scoped to certain directories (using the `files` option, with glob matching).

Reply:
No, Im not. As explained before pieces of the legacy-html generator have web-targeted aasets, such as /assets/api.js

ESLint complains about all Browser-APIs there because lack of target.

Reply:
I know v8 supports that, but does v9? Any example of it? I can try to Google it too

Reply:
https://eslint.org/docs/latest/use/configure/configuration-files#specifying-files-and-ignores

Later in this array, add:

```
{
  files: ['src/generators/legacy-html/assets/*.js'],
  languageOptions: { globals: { ...globals.browser } }
},
```